hooks/queries/rewards/useOutstandingRewards.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from "@tanstack/react-query";
+import { outstandingRewardsKey } from "constant";
+import { BigNumber } from "ethers";
+import { useAccountDetails, useContractsContext, useHandleError } from "hooks";
+import { getOutstandingRewards } from "web3";
+
+export function useOutstandingRewards() {
+  const { voting } = useContractsContext();
+  const { address } = useAccountDetails();
+  const { onError } = useHandleError({ isDataFetching: true });
+
+  const queryResult = useQuery(
+    [outstandingRewardsKey, address],
```

Comment:
build is erroring because this `outstandingRewardsKey` doesn't exist. Did you add this to the `queryKeys.ts` file in the `constant` dir?